fl_training.py

```diff
@@ -121,6 +122,7 @@ def compute_test_score(
     x_test,
     y_test,
     is_early_stopping=True,
+    aggregation_weights=None,
```

Comment:
I think we should have a param in the scenario class to choose which aggregation we want to perform (for now it will only be: weighted by data size or weighted by nothing)
It will also be easier to test the impact of the weighted average, so we can be sure that weighted average is better than no weight.

Reply:
Ok, done in 6b0ec01.